Formula/qmmp.rb

```diff
@@ -0,0 +1,38 @@
+class Qmmp < Formula
+  desc "Qt-based Multimedia Player"
+  homepage "http://qmmp.ylsoftware.com"
+  url "http://qmmp.ylsoftware.com/files/qmmp-1.1.8.tar.bz2"
+  sha256 "17bc88d00ea0753e6fc7273592e894320f05cae807f7cc2c6a5351c73217f010"
+
+  depends_on "cmake" => :build
+  depends_on "qt"
+  depends_on "ffmpeg"
+  depends_on "flac"
+  depends_on "libvorbis"
+  depends_on "libmad"
+  depends_on "libogg"
+  depends_on "libsoxr"
+  depends_on "faad2"
+  depends_on "opus"
+  depends_on "opusfile"
+  depends_on "mplayer"
+  depends_on "libbs2b"
+  depends_on "libsndfile"
+  depends_on "musepack"
+  depends_on "taglib"
+  depends_on "libmms"
+  depends_on "libsamplerate"
+
+  head do
+    url "http://svn.code.sf.net/p/qmmp-dev/code/branches/qmmp-1.2/"
+  end
```

Comment:
Use `head "http://svn.code.sf.net/p/qmmp-dev/code/branches/qmmp-1.2/"` at the top of the file instead.